app/main.py

```diff
@@ -1 +1,21 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    mv_command, from_file, destination_location = command.split()
+
+    if mv_command != "mv":
+        raise Exception('Command should start with "mv"!')
+
+    *directories, to_file = destination_location.split("/")
+    directory_path = ""
+
+    for directory in directories:
+        directory_path = os.path.join(directory_path, directory)
+        os.mkdir(directory_path)
```

Comment:
try to use `os.path.split(destination_location)` and `os.makedirs() ` to remove the loop